pygmt/base_plotting.py

```diff
@@ -440,6 +440,187 @@ def grdview(self, grid, **kwargs):
                 arg_str = " ".join([fname, build_arg_string(kwargs)])
                 lib.call_module("grdview", arg_str)
 
+    @fmt_docstring
+    @use_alias(
+        R="region",
+        J="projection",
+        B="frame",
+        C="cmap",
+        U="timestamp",
+    )
+    @kwargs_to_strings(R="sequence", i="sequence_comma")
+    def velo(self, data=None, scaling=None, **kwargs):
+        """
```

Comment:
> I didnt find any other way than using '\ ' to be able to have different formating in the same word.
> The idea is to reproduce the [original gmt formatting of the argument](https://docs.generic-mapping-tools.org/latest/supplements/geodesy/velo.html?highlight=velo#required-arguments) :
> **-Se**_velscale/confidence_[**+f**_font_]

You're actually did very well, I tried using the solution at https://meta.stackexchange.com/questions/183610/how-to-combine-bold-and-code-sample-in-markdown but it's not as elegant as yours.

Anyhow, the solution to silence the ["Invalid escape sequence"](https://www.flake8rules.com/rules/W605.html) error is surprisingly simple - [use a raw string](https://stackoverflow.com/questions/61313624/docstring-invalid-escape-sequence-triggered-by-a-display-equation-in-latex#comment108467285_61313624)!

```suggestion
        r"""
```